libraries/aws/shadow/cbmc/proofs/Shadow_GetTopicString/README.md

```diff
@@ -0,0 +1,10 @@
+Shadow_GetTopicString proof
+==============
+
+This directory contains a memory safety proof for Shadow_GetTopicString.
```

Comment:
- Can you add a note about expected runtime and coverage?
So this is the place I could note about expected runtime and coverage? how do I describe the coverage? paste the result from html/index.html? since there are no example so far.
And I just curious what does your expected runtime stand for? 
- Were any functions stubbed out?
No, what kind of function should be stubbed out?

Reply:
I have reduced the run time on Shadow_MatchTopic to instant and Shadow_GetTopicString is also very fast. No functions were needed to be stubbed out. Both of these proofs have 100% coverage. It is not typical for us to edit this README.md for information found in actually running the proof. Could I please I request that we do not add this information?

Reply:
It doesn't have to be on the README.  But its good practice to track what the runtime/coverage was when the proof was finished, as well as any caveats about the proof, so that future maintainers have that information readily available.